src/cyhy_cvesync/cve_sync.py

```diff
@@ -68,20 +68,35 @@ async def process_cve_json(cve_json: dict) -> Tuple[int, int]:
             raise ValueError("CVE ID is empty.")
 
         # Only process CVEs that have CVSS V2 or V3 data
-        if any(k in cve["impact"] for k in ["baseMetricV2", "baseMetricV3"]):
+        if any(
+            k in cve["cve"].get("metrics", {})
+            for k in [
+                "cvssMetricV2",
+                "cvssMetricV30",
+                "cvssMetricV31",
+            ]
+        ):
             # Check if the CVE document already exists in the database
             global cve_map
             async with cve_map_lock:
                 cve_doc = cve_map.pop(cve_id, None)
 
-            version = "V3" if "baseMetricV3" in cve["impact"] else "V2"
+            # Determine newest CVSS metrics version in the CVE data
+            metrics_version = None
+            for v in ["cvssMetricV31", "cvssMetricV30", "cvssMetricV2"]:
```

Comment:
Just wanted to mention https://github.com/cisagov/cyhy-core/pull/103#discussion_r2299036039 in this pull request as well.

Reply:
Please take a look at the changes made in https://github.com/cisagov/cyhy-cvesync/pull/10/commits/042aedb1cf28875c28533e2128185ac8200c7faa that are similar to those in https://github.com/cisagov/cyhy-core/commit/95bd787be1baf1b0987fb2b7c05bb290f57cc4d7.